graylog2-server/src/main/java/org/graylog2/shared/security/tls/SelfSignedCertificate.java

```diff
@@ -16,29 +16,28 @@
  */
 package org.graylog2.shared.security.tls;
 
+import org.apache.commons.io.IOUtils;
+import org.bouncycastle.asn1.ASN1InputStream;
+import org.bouncycastle.asn1.ASN1Sequence;
+import org.bouncycastle.asn1.x500.X500Name;
+import org.bouncycastle.asn1.x509.*;
```

Comment:
Please don't use wildcard imports.

If you're using IntelliJ IDEA, here are instructions how to (almost) disable wildcard imports: http://stackoverflow.com/a/3348855